kubectl SIGSEGV if k8s not running

### What Happened?

I'm just getting going with Kubernetes, so trying out minikube.  I was in the middle of downloading my first instance with:
```sh
$ minikube start --driver=docker
😄  minikube v1.25.1 on Ubuntu 20.04
✨  Using the docker driver based on user configuration
👍  Starting control plane node minikube in cluster minikube
🚜  Pulling base image ...
💾  Downloading Kubernetes v1.23.1 preload ...
    > gcr.io/k8s-minikube/kicbase: 378.98 MiB / 378.98 MiB  100.00% 76.80 KiB p
    > preloaded-images-k8s-v16-v1...: 128.42 MiB / 504.42 MiB  25.46% 88.93 Ki
```

Meanwhile, in another shell I attempted to list.  Bad move apparently:
```sh
$ minikube kubectl -- get pods -A
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x4a8 pc=0x1a60f12]

goroutine 1 [running]:
k8s.io/minikube/cmd/minikube/cmd.glob..func23(0x3530580, {0xc000acc800, 0x3, 0x4})
	/app/cmd/minikube/cmd/kubectl.go:102 +0x8d2
github.com/spf13/cobra.(*Command).execute(0x3530580, {0xc000acc7c0, 0x4, 0x4})
	/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:860 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0x3532380)
	/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.3.0/command.go:902
k8s.io/minikube/cmd/minikube/cmd.Execute()
	/app/cmd/minikube/cmd/root.go:157 +0xa31
main.main()
	/app/cmd/minikube/main.go:86 +0x255
```

This worked _after_ the download finished, but a segfault is, well, a segfault.  :-/

### Attach the log file

```sh
$ minikube logs --file=log.txt
[nothing written here, and empty log file]
```

### Operating System

Ubuntu

### Driver

Docker

--- a/cmd/minikube/cmd/kubectl.go
+++ b/cmd/minikube/cmd/kubectl.go
@@ -56,8 +56,10 @@ host. Please be aware that when using --ssh all paths will apply to the remote m
 		cc, err := config.Load(ClusterFlagValue())
 
 		version := constants.DefaultKubernetesVersion
+		binaryMirror := ""
 		if err == nil {
 			version = cc.KubernetesConfig.KubernetesVersion
+			binaryMirror = cc.BinaryMirror
 		}
 
 		cname := ClusterFlagValue()
@@ -99,7 +101,7 @@ host. Please be aware that when using --ssh all paths will apply to the remote m
 			args = append(cluster, args...)
 		}
 
-		c, err := KubectlCommand(version, cc.BinaryMirror, args...)
+		c, err := KubectlCommand(version, binaryMirror, args...)
 		if err != nil {
 			out.ErrLn("Error caching kubectl: %v", err)
 			os.Exit(1)

